lib/glific/providers/gupshup_enterprise/response_handler.ex

```diff
@@ -41,7 +41,16 @@ defmodule Glific.Providers.Gupshup.Enterprise.ResponseHandler do
     end
   end
 
-  @spec add_error_payload(Tesla.Env.t()) :: Tesla.Env.t()
+  # Sending default error when API Client call fails for some reason
+  def handle_response(_error, message) do
```

Comment:
@AkhileshNegi I did not get the code completely but can't we use the default tesla error here also?